docs/Configuration.md

```diff
@@ -1,13 +1,15 @@
 # Configuration
 
-# The following parameters must be set on the AdyenCheckout main instance.
-
-### Root configurations
+## Root configurations
 * **environment** - Use `test`. Project is currently in **BETA** stage; to do penny-test, change the value to one of our [live environments](https://docs.adyen.com/online-payments/drop-in-web#testing-your-integration).
 * **clientKey** - A public key linked to your API credential, used for [client-side authentication](https://docs.adyen.com/development-resources/client-side-authentication).
-* **amount** - Amount to be displayed on the Pay Button. It expects an object with the minor units value and currency properties. For example, { value: 1000, currency: 'USD' }. For BIN or card verification requests, set amount to 0 (zero).
-* **countryCode** - The shopper's country code in ISO 3166-1 alpha-2 format. Example: NL or DE.
-* **shopperLocale** - 🚧 Work in progress. In current version this property only localises payment methods names. Default OS's locale is used for localisation. You can override  
+* **amount** - Amount to be displayed on the Pay Button. It expects an object with the minor units value and currency properties. For example, { value: 1000, currency: 'USD' }. For card pre-authorisation set amount to 0 (zero).
+* **countryCode** - The shopper's country code in ISO 3166-1 alpha-2 format. Example: NL or DE. 
+* **shopperLocale** - 🚧 Work in progress. In current version this property only localises payment methods names. Default OS's locale is used for localisation.
+* **returnUrl** - For iOS, this is the URL to your app, where the shopper should return, after a redirection. Maximum of 1024 characters. For more information on setting a custom URL scheme for your app, read the [Apple Developer documentation](https://developer.apple.com/documentation/uikit/inter-process_communication/allowing_apps_and_websites_to_link_to_your_content/defining_a_custom_url_scheme_for_your_app).
+For Android, this value is automatically overridden by AdyenCheckout.
```

Comment:
Style guide and consistency with Docs suggestion: also `monospace` for class/file names: `AdyenCheckout`

Reply:
Fixed [d53a296](https://github.com/Adyen/adyen-react-native/pull/146/commits/d53a296032f048de0fec4cc5aeeed235e3755a7e)